catroid/src/main/res/values-ta/strings.xml

```diff
@@ -1736,7 +1736,7 @@
     <string name="formula_editor_logic_comparison">ஒப்பீட்டு ஆபரேட்டர்கள்</string>
     <string name="formula_editor_object_general">பொது பண்புகள்</string>
     <string name="formula_editor_object_movement">இயக்க பண்புகள்</string>
-    <string name="formula_editor_device_visual_detection">Visual sensors</string>
+    <string name="formula_editor_device_face_detection">Visual sensors</string>
```

Comment:
undo this changes, nothing to do here, it will eventually be overwritten by the translations